server/scripts/seed.ts

```diff
@@ -0,0 +1,36 @@
+import Db from 'server/lib/db';
+import log from 'server/lib/log';
+import User from 'server/models/User.model';
+import { makeUser } from 'server/test/utils';
+import { UserObject } from 'server/types/User';
+
+async function create100Users(): Promise<UserObject[]> {
```

Comment:
I think this would actually be pretty useful as a test utility script as "create many users", which allows you to pass an arbitrary number of users to create, then to just call that function here, passing 100.